.travis.yml

```diff
@@ -23,7 +23,6 @@ before_install:
   - go get -u github.com/client9/misspell/cmd/misspell
   - go get -u golang.org/x/lint/golint
   - go get github.com/fzipp/gocyclo
-  - go get -u honnef.co/go/tools/cmd/megacheck
```

Comment:
Should we replace this with staticcheck?

Reply:
Yes I'll do that!